api/common.go

```diff
@@ -27,12 +27,26 @@ const (
 	DefaultDockerfileName string = "Dockerfile"
 )
 
-// byPrivatePort is temporary type used to sort types.Port by PrivatePort
-type byPrivatePort []types.Port
+// portOrdering is a temporary type used to sort types.Port by its fields
+type portOrdering []types.Port
```

Comment:
mmm can we call this something like `alphabetically` or something like this? @duglin do you have any other idea?